lib/EmitterInterface.php

```diff
@@ -46,6 +46,8 @@ public function once(string $eventName, callable $callBack, int $priority = 100)
      *
      * Lastly, if there are 5 event handlers for an event. The continueCallback
      * will be called at most 4 times.
+     *
+     * @param array<int, mixed> $arguments
```

Comment:
i think we could type the callback signature to
```suggestion
     * @param array<int, mixed> $arguments
     * @param callable():bool $continueCallBack
```

Reply:
yes, this works:
```
@param callable():bool|null $continueCallBack
```